### ISSUE TITLE
Add package-lock.json and yarn.lock

### ISSUE THREAD

**Issue by TimDaub** [austintgriffith/burner-wallet#216]
## Scope
- burner-wallet and potentially other components like dapparatus currently do not include package-lock.json or yarn.lock
- These files are supposed to be committed however as they provide integrity when installing dependencies

## Deliverables
- Add yarn.lock and package-lock.json in all components used by the burner-wallet.

## Roles
bounty gardener: @TimDaub / 15%
bounty worker: name / 75%
bounty reviewer: name / 15%

**Comment by johannbarbie** [comment]
*by pedro:*
You shouldnt include both lock files
You should use one or the other
Otherwise you dont have integrity between people developing with yarn and npm
Unless you include a pre-push script to update both for consistency
And even then they wouldnt always result the same

**Comment by dmihal** [comment]
Agreed, in todays call we seemed to agree to use Yarn and yarn.lock.

**Comment by kashaiahyah85** [comment]
The npm package-lock.json should never be committed; the yarn.lock is meant to be. If you use yarn; it's best to alter the package.json to include a requirement for it. Either both must go, or make the good call to hop on to Yarn all the way.

**Comment by yanyishuai** [comment]
Submitted PR for #216 — adds package-lock.json (yarn.lock already on master).

**Comment by rovidev95** [comment]
Delivered in PR https://github.com/austintgriffith/burner-wallet/pull/311

- Removed `package-lock.json` from `.gitignore`
- Added lockfile generated via `npm install --package-lock-only --legacy-peer-deps`
- CI installs with `--legacy-peer-deps` for reproducible builds

yarn.lock for sub-packages (dapparatus) would need separate PRs in those repos.

/claim #216 rovidev95@gmail.com

**Comment by rovidev95** [comment]
Split PR for incremental review: **#314** — /claim rovidev95@gmail.com in PR body. @TimDaub
